<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ajoutDossier</class>
 <widget class="QDialog" name="ajoutDossier">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>824</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Enregistrer Dossier</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Client</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Repertoire</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Date Reception</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Arrivé Bateau</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="clientComboBox"/>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="repEdit"/>
           </item>
           <item row="2" column="1">
            <widget class="QDateEdit" name="recepdateEdit">
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QDateEdit" name="arrdateEdit">
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="btModif">
             <property name="text">
              <string>Modifier</string>
             </property>
             <property name="icon">
              <iconset resource="../Res.qrc">
               <normaloff>:/icones/Icones/05-Edit-icon.png</normaloff>:/icones/Icones/05-Edit-icon.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btValider">
             <property name="text">
              <string>Valider</string>
             </property>
             <property name="icon">
              <iconset resource="../Res.qrc">
               <normaloff>:/icones/Icones/15-Tick-icon.png</normaloff>:/icones/Icones/15-Tick-icon.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btAnnuler">
             <property name="text">
              <string>Annuler</string>
             </property>
             <property name="icon">
              <iconset resource="../Res.qrc">
               <normaloff>:/icones/Icones/19-Left-Arrow-icon.png</normaloff>:/icones/Icones/19-Left-Arrow-icon.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Liste des conteneurs du dossier</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QTableView" name="contTableView">
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::SingleSelection</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="btModifier">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Modifier</string>
              </property>
              <property name="icon">
               <iconset resource="../Res.qrc">
                <normaloff>:/icones/Icones/05-Edit-icon.png</normaloff>:/icones/Icones/05-Edit-icon.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btSupprimer">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Supprimer</string>
              </property>
              <property name="icon">
               <iconset resource="../Res.qrc">
                <normaloff>:/icones/Icones/14-Delete-icon.png</normaloff>:/icones/Icones/14-Delete-icon.png</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Ajouter conteneur</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QFormLayout" name="formLayout_2">
            <item row="0" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Numéro</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Marchandise</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Type</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QComboBox" name="typeComboBox">
              <item>
               <property name="text">
                <string>20 Pieds</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>40 Pieds</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="marchandiseEdit"/>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="numeroEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="btAjouter">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Ajouter</string>
              </property>
              <property name="icon">
               <iconset resource="../Res.qrc">
                <normaloff>:/icones/Icones/add.ico</normaloff>:/icones/Icones/add.ico</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btValide">
              <property name="text">
               <string>Valider</string>
              </property>
              <property name="icon">
               <iconset resource="../Res.qrc">
                <normaloff>:/icones/Icones/15-Tick-icon.png</normaloff>:/icones/Icones/15-Tick-icon.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btEffacer">
              <property name="text">
               <string>Annuler</string>
              </property>
              <property name="icon">
               <iconset resource="../Res.qrc">
                <normaloff>:/icones/Icones/19-Left-Arrow-icon.png</normaloff>:/icones/Icones/19-Left-Arrow-icon.png</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btFermer">
       <property name="text">
        <string>Fermer</string>
       </property>
       <property name="icon">
        <iconset resource="../Res.qrc">
         <normaloff>:/icones/Icones/gnome_application_exit.png</normaloff>:/icones/Icones/gnome_application_exit.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>clientComboBox</tabstop>
  <tabstop>repEdit</tabstop>
  <tabstop>recepdateEdit</tabstop>
  <tabstop>arrdateEdit</tabstop>
  <tabstop>btValider</tabstop>
  <tabstop>btAnnuler</tabstop>
  <tabstop>contTableView</tabstop>
  <tabstop>btModifier</tabstop>
  <tabstop>btSupprimer</tabstop>
  <tabstop>numeroEdit</tabstop>
  <tabstop>marchandiseEdit</tabstop>
  <tabstop>typeComboBox</tabstop>
  <tabstop>btAjouter</tabstop>
  <tabstop>btValide</tabstop>
  <tabstop>btEffacer</tabstop>
  <tabstop>btFermer</tabstop>
 </tabstops>
 <resources>
  <include location="../Res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btAnnuler</sender>
   <signal>clicked()</signal>
   <receiver>repEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>342</x>
     <y>125</y>
    </hint>
    <hint type="destinationlabel">
     <x>318</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btAnnuler</sender>
   <signal>clicked()</signal>
   <receiver>clientComboBox</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>335</x>
     <y>125</y>
    </hint>
    <hint type="destinationlabel">
     <x>295</x>
     <y>25</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btEffacer</sender>
   <signal>clicked()</signal>
   <receiver>marchandiseEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>723</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>704</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btEffacer</sender>
   <signal>clicked()</signal>
   <receiver>numeroEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>703</x>
     <y>125</y>
    </hint>
    <hint type="destinationlabel">
     <x>614</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btFermer</sender>
   <signal>clicked()</signal>
   <receiver>ajoutDossier</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>726</x>
     <y>526</y>
    </hint>
    <hint type="destinationlabel">
     <x>604</x>
     <y>337</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
